core/mondoo-azure-security.mql.yaml

```diff
@@ -1223,7 +1229,13 @@ queries:
     severity: 80
     docs:
       desc: |
-        Azure Key Vault can store keys, secrets, and certificates. Accidental unavailability of a Key Vault can cause serious problems in any organization. 
+        Azure Key Vault can store keys, secrets, and certificates. Accidental unavailability of a Key Vault can cause serious problems in any organization.
```

Comment:
What do we mean by accidental unavailability? Is that just someone accidentally deleting it?

Reply:
exactly, by accident deleting it!